requirements.txt

```diff
@@ -1,7 +1,11 @@
-click
-elasticsearch
-flask
-flask_cors
-pyyaml
-requests
-shapely
+elasticsearch==6.2.0
```

Comment:
Can we unpin deps? As well:
- move pytest to requirements-dev.txt
- remove setuptools
- do we need pypandoc?
- jinja2 comes with Flask by default; can we remove?

Reply:
requirements.txt was autogenerated with pipreqs (https://github.com/bndr/pipreqs)